fabtests/unit/cq_test.c

```diff
@@ -171,6 +171,8 @@ cq_signal()
 		sprintf(err_buf, "fi_cq_open(1, 0, FI_CQ_FORMAT_UNSPEC, "
 				"FI_WAIT_UNSPEC) = %d, %s",
 				ret, fi_strerror(-ret));
+		if (ret == -FI_EOPNOTSUPP)
```

Comment:
Not sure why we need this change, before your PR, EFA cq won't support WAIT_UNSPEC anyway right?

Reply:
I think we should use the same errno for device unsupported case as well

Reply:
>FI_EOPNOTSUPP
    Operation not supported on transport endpoint

Which is not exactly what we have